dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/ExportBaseCommand.java

```diff
@@ -63,6 +64,7 @@
 import org.apache.camel.util.FileUtil;
 import org.apache.camel.util.IOHelper;
 import org.apache.camel.util.StringHelper;
+import org.apache.commons.io.IOUtils;
```

Comment:
Can you avoid using this library and use what we have in Camel already - There is something likely already on IOHelper or elsewhere in Camel to copy an InputStream

Reply:
Done